src/libs/elektra/exportsymbols.c.in

```diff
@@ -9,6 +9,22 @@
 #include <stdio.h>
 #include <string.h>
 
+// clang-format off
+static const char * excludedPlugins[] = { @SHARED_ONLY_PLUGINS_ARRAY@ };
```

Comment:
I am afraid that won't work for cross-compilation. Why not simply remove the "SHARED_ONLY_PLUGINS" on CMake level (src/libs/elektra/CMakeLists.txt line 25 and 29) and simply not pass them as arguments to exportsymbols.